### ISSUE TITLE
Move code towards functions or classes?

### ISSUE THREAD

**Issue by looselycoupled** [rebeccabilbro/titanic#3]
We usually want to move those new to python towards always organizing their code into functions or classes.  Is there a way we can restructure titanic so it's less a series of python statements and more a series of functions that perform specific needs?

**Comment by looselycoupled** [comment]
We could also include an

```
if __name__ == '__main__': 
```

which then calls each function (which start out empty).  This helps enforce the idea of creating a module that can be reused by other Python files.

**Comment by rebeccabilbro** [comment]
@looselycoupled I thought about this comment a lot, and ultimately decided that this lab is conceived as more of a dialogic exploration than a systematic wrangling tool. I agree with you that moving them towards encapsulation is important, but this lab really felt like it should be an iPython notebook. So I've changed it over. I did address all of the other comments you provided in the issues, so I hope this is okay. After Saturday, when we have some more time, maybe we can discuss how this might be restructured towards a more object-oriented approach for future semesters. Thanks again for your help.
